setup.py

```diff
@@ -14,17 +14,43 @@
     install_requires+=['ordereddict','unittest2']
 
 
+def embed_version(basepath, reponame, ref='v0.2.1'):
+    """
+    Autover is purely a build time dependency in all cases (conda and
+    pip) except for when you use pip's remote git support [git+url] as
+    1) you need a dynamically changing version and 2) the environment
+    starts off clean with zero dependencies installed.
+
+    This function acts as a fallback to make Version available until
+    PEP518 is commonly supported by pip to express build dependencies.
+    """
+    import io, zipfile
+    try:    from urllib.request import urlopen
+    except: from urllib import urlopen
+    response = urlopen('https://github.com/ioam/autover/archive/{ref}.zip'.format(ref=ref))
+    zf = zipfile.ZipFile(io.BytesIO(response.read()))
+    ref = ref[1:] if ref.startswith('v') else ref
+    embed_version = zf.read('autover-{ref}/autover/version.py'.format(ref=ref))
+    with open(os.path.join(basepath, reponame, 'version.py'), 'wb') as f:
+        f.write(embed_version)
+
+
 def get_setup_version(reponame):
     """
     Helper to get the current version from either git describe or the
     .version file (if available).
     """
     basepath = os.path.split(__file__)[0]
     version_file_path = os.path.join(basepath, reponame, '.version')
-    try:
-        version = importlib.import_module(reponame + ".version") # Bundled
-    except:
-        version = None
```

Comment:
I'll just note that the shortened version looks fine! We can return to it later but for now, it is safest to stick to what is properly tested in autover.